### PR TITLE
Reviews routes#18

### PR DESCRIPTION
**Pull request by Gabbyjose**
Created review api folder with the routes outlined in the ticket. I have not yet required this into the index file because i wanted to wait until all routes were done before editing index

### CODE REVIEW

File: server/api/reviews.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  Review.findAll({
+    where: {
+      productId: req.params.prodId
+    }
+  })
+  .then(review => res.json(review))
+  .catch(next)
+})
+
+//i realized that we wouldn't post to '/:prodId' like we have in our ticket, we would just post to '/' because we don't have a product ID at time of creation
```
Comment (satsivi) [inline server/api/reviews.js]: I think this is correct too!

File: server/api/reviews.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const router = require('express').Router();
+const {Review} = require('../db/models');
+
+router.get('/:prodId', (req, res, next) => {
+  Review.findAll({
```
Comment (satsivi) [inline server/api/reviews.js]: can use findById instead

Comment (Gabbyjose) [reply server/api/reviews.js]: you're right!

File: server/api/reviews.js (hunk 1 above)
Comment (Gabbyjose) [reply server/api/reviews.js]: cool

**Pull request merged**